Public/Src/Cache/ContentStore/Distributed/Utilities/GrpcDistributedPathTransformer.cs

```diff
@@ -50,31 +51,30 @@ public byte[] GetLocalMachineLocation(AbsolutePath cacheRoot)
                 cacheRoot = cacheRoot / Constants.SharedDirectoryName;
             }
 
-            var cacheRootString = cacheRoot.Path.ToUpperInvariant();
+            var cacheRootString = cacheRoot.Path.PathToUpperInvariant();

```

Comment:
I'm curious, why not to add a similar extension method to AbsolutePath itself, instead of adding yet another extension method to string?

I think it'll look better and the solution won't pollute an intellisense:

`cacheRoot.ToUpperInvariant()`?

Reply:
Why not an actual method? Adding methods isn't breaking.